tox.ini

```diff
@@ -230,6 +230,7 @@ deps =
     psycopg225: psycopg2>=2.5,<2.6
     psycopg226: psycopg2>=2.6,<2.7
     psycopg227: psycopg2>=2.7,<2.8
+    psycopg227b: psycopg2==2.7-beta2
```

Comment:
this will probably fail, since you have `27-beta` above, but `227b` here.

We probably don't need to test beta versions, the feedback was just about making our version parsing more protective to beta versions.

We don't need to test beta since we don't want to "support" beta versions.